openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/SWIGResources.java

```diff
@@ -76,19 +82,41 @@
      */
     private Integer boosterNumIterations;
 
+    /**
+     * Number of features in the trained LightGBM boosting model.
+     * Whilst not a swig resource, it is automatically retrieved during model loading,
+     * thus we store it to avoid calls that can fail.
+     */
+    private Integer boosterNumFeatures;
+
     /**
      * Constructor. Initializes a model handle and all resource handlers
      * necessary to implement the standard model operations.
      *
      * @param modelPath         Path to the model folder.
-     * @param numFeaturesSchema Number of features in the model.
      * @throws ModelLoadingException Error loading the model.
      * @throws LightGBMException     in case there's an error in the C++ core library.
      */
-    SWIGResources(final String modelPath, final int numFeaturesSchema) throws ModelLoadingException, LightGBMException {
+    SWIGResources(final String modelPath,
+                  final String LightGBMParameters) throws ModelLoadingException, LightGBMException {
```

Comment:
Javadoc is changed to remove the previous parameter, but you should document the new one